content/v3/csidriver/troubleshooting/unity.md

```diff
@@ -16,3 +16,4 @@ description: Troubleshooting Unity XT Driver
 | When a node goes down, the block volumes attached to the node cannot be attached to another node | 1. Force delete the pod running on the node that went down <br /> 2. Delete the VolumeAttachment to the node that went down. <br /> Now the volume can be attached to the new node. |
 | Standby controller pod is in crashloopbackoff state | Scale down the replica count of the controller pod's deployment to 1 using ```kubectl scale deployment <deployment_name> --replicas=1 -n <driver_namespace>``` |
 | fsGroupPolicy may not work as expected without root privileges for NFS only [https://github.com/kubernetes/examples/issues/260](https://github.com/kubernetes/examples/issues/260) | To get the desired behavior set “RootClientEnabled” = “true” in the storage class parameter |
+| When both iSCSI IQNs and FC WWNs are present, host registrations on Unity systems will include all initiators. If only FC WWNs are present, a warning message will appear: **‘iSCSI Initiators’ cannot be retrieved.**| Ensure only desired initiators are configured on the worker nodes to limit the initiators included in the host registrations |
```

Comment:
where the warning will be displayed? can you attach the screenshot?

Reply:
In node pod logs when it tries to add the initiators : https://github.com/dell/csi-unity/blob/main/service/node.go#L1524
@rajkumar-palani